### ISSUE TITLE
InvalidArchiveError("Error with archive ....)

### ISSUE THREAD

**Issue by paripudi** [conda/conda#9142]
The below issue occurs while trying to install anacondo directly or by installin it through mini conda.

Upon executing the script: `conda install anaconda`, the below error appeared:

```
InvalidArchiveError("Error with archive C:\\Users\\<username>\\AppData\\Local\\Continuum\\miniconda3\\pkgs\\libtiff-4.0.10-hb898794_2.conda.  You probably need to delete and re-download or re-create this file.  Message from libarchive was:\n\nCan't create '\\\\?\\C:\\Users\\<username>\\AppData\\Local\\Continuum\\miniconda3\\pkgs\\libtiff-4.0.10-hb898794_2\\Library\\bin\\tiff2pdf.exe' (errno=2, retcode=-25, archive_p=2811869718192)")
```

**Comment by jakirkham** [comment]
Yeah I ran into this on a Linux box recently as well. I wonder if instead of throwing this error Conda could delete the tarball and redownload (perhaps issuing a warning along the way).

**Comment by jakirkham** [comment]
Could we please reopen this one?

**Comment by tferguson47** [comment]
Sharing some increased reporting of InvalidArchiveError on Windows (Jan-Feb 2025)

We're seeing a pattern of InvalidArchiveError reports from Windows users that aren't resolved by standard troubleshooting steps. This appears to be affecting multiple packages and user configurations.
Recent Examples

Case 1: jupyterlab_widgets installation

User: https://forum.anaconda.com/t/conda-install-jupyter-fails/108348/4
Error: InvalidArchiveError("Error with archive C:\Users\dskga\anaconda3\pkgs\jupyterlab_widgets-3.0.16-py313haa95532_0.co)
Command: conda create --name Physics python=3.13 numpy scipy sympy plotly jupyter
Environment: Windows, Anaconda distribution

Case 2: PyTorch installation

User: https://forum.anaconda.com/t/pytorch-installation-for-conda/108336/3
Error: InvalidArchiveError("Error with archive C:\Users\jwb20148\AppData\Local\anaconda3\pkgs\pytorch-2.7.0-gpu_cuda128...
Command: conda install pytorch-gpu -c defaults
Environment: Windows 11, Anaconda x86_64 distribution
Note: Error persisted across multiple fresh environments

This user attempted to install from conda-forge instead and things worked fine. The packaging team was unable to recreate this issue.

Troubleshooting Attempted (No Resolution)
Users tried standard remediation without success:
conda clean --all (or conda clean -ay)
Creating fresh environments
Deleting and recreating environments with conda remove --all -n <env>

**Comment by danyeaw** [comment]
I think right now if the extraction fails, the directory would have already been created and it may not yet have an `info/` folder. If this happens, then `clean --all` would skip it.

I think to fix this we would need a few changes in conda:

- Cleanup folders even if they have no info folder
- Ensure that `ExtractPackageAction.reverse()` happens rather than directly exiting during a package extraction failure

**Comment by SuzannevanNoordt** [comment]
Hi all,

Is there any new information on this error. I get the same error for updating jupyterlab_widgets, and for installing pygimli, a geophysics modeling software. Pygimli relies on viskores which has to be extracted using conda-forge.

I describe the steps taken in detail here: https://github.com/gimli-org/pyGIMLi/issues/948

Thanks for your time!

**Comment by PaavoHietala** [comment]
I've encountered the same error with the viskores package while installing MNE-Python with Conda.

Following the guide at https://mne.tools/stable/install/manual_install.html and running the following command

`conda create --channel=conda-forge --strict-channel-priority --name=mne mne` 

in Powershell throws `InvalidArchiveError`. With two Windows machines the problem was with the Windows default `MAX_PATH` limit of 260 characters. Enabling long paths with a registry entry `LongPathsEnabled` fixed the issue.

The registry entry can be created in PowerShell (run as administrator) with the following command (or manually via Registry editor)

`New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force`

**Comment by SuzannevanNoordt** [comment]
> I've encountered the same error with the viskores package while installing MNE-Python with Conda.
> 
> Following the guide at https://mne.tools/stable/install/manual_install.html and running the following command
> 
> `conda create --channel=conda-forge --strict-channel-priority --name=mne mne`
> 
> in Powershell throws `InvalidArchiveError`. With two Windows machines the problem was with the Windows default `MAX_PATH` limit of 260 characters. Enabling long paths with a registry entry `LongPathsEnabled` fixed the issue.
> 
> The registry entry can be created in PowerShell (run as administrator) with the following command (or manually via Registry editor)
> 
> `New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force`

Thanks for this! I saw that fix being applied elsewhere but not sure how to run as administrator. I will figure it out and hopefully should work then!

**Comment by ghylander** [comment]
I too had this issue with the viskores package and enabling longpaths solved it, thanks!

**Comment by travishathaway** [comment]
Before we work on this, can we please update the description with clear acceptance criteria.
